### PR TITLE
Revert strategies/exit.py (#89) as observing slow exit_by_trailing_stop.

### PR DESCRIPTION
**Pull request by hiohiohio**
exit_by_trailing_stop with market data for a day becomes slower than before.
Sorry for no benchmark results attached.

### CODE REVIEW

File: src/backlight/strategies/exit.py [hunk 1]
```diff
@@ -80,10 +80,8 @@ def _exit(
 
         df = pd.DataFrame(index=indices, data=exits, columns=["amount", "_id"])
 
-        return from_dataframe(df, symbol, currency_unit)
+        return from_dataframe(df, entries.symbol, trades.currency_unit)
```
Comment (keisuke-umezawa) [inline src/backlight/strategies/exit.py]: `from_dataframe(df, trades.symbol, trades.currency_unit)`?

File: src/backlight/strategies/exit.py [hunk 2]
```diff
@@ -124,21 +122,14 @@ def _exit_by_max_holding_time(
...
-            df_exit = df.loc[start:end]
+            df_exit = df[(idx <= df.index) & (df.index <= idx + max_holding_time)]
             transaction = _exit_transaction(df_exit, trade, exit_condition)
-
             indices.append(transaction.timestamp)
             exits.append((transaction.amount, i))
 
         df = pd.DataFrame(index=indices, data=exits, columns=["amount", "_id"])
-        return from_dataframe(df, symbol, currency_unit)
+        return from_dataframe(df, entries.symbol, trades.currency_unit)
```
Comment (keisuke-umezawa) [inline src/backlight/strategies/exit.py]: `return from_dataframe(df, trades.symbol, trades.currency_unit)`?

File: src/backlight/strategies/exit.py (hunk 1 above)
Comment (hiohiohio) [reply src/backlight/strategies/exit.py]: Oh, yes. Thank you.

File: src/backlight/strategies/exit.py (hunk 2 above)
Comment (hiohiohio) [reply src/backlight/strategies/exit.py]: fixed.

**Pull request merged**